website/content/docs/agent/index.mdx

```diff
@@ -186,6 +186,9 @@ These are the currently-available general configuration options:
 
 - `listener` <code>([listener][listener]: <optional\>)</code> - Specifies the addresses and ports on which the Agent will respond to requests.
 
+  ~> **Note:** On `SIGHUP` (`sudo kill -s HUP {PID of vault}`), Vault Agent will attempt to reload listener TLS configuration.
```

Comment:
I have a slight preference to remove the `sudo` from here, as this can be done without sudo and I'd err on the side of recommending least privilege. If it's there for symmetry with the SIGHUP docs for Vault itself, that's a little different, I guess, but yeah.

Reply:
Yeah, I just tried to match the command we show for the Vault server (https://developer.hashicorp.com/vault/docs/configuration#log_level) but can remove it for Agent easily enough :)

Reply:
Yeah, I guess I'm inclined to go low privilege where possible, I'd rather we didn't have `sudo` in the other docs either, but I'll admit that this is a bit of a nit.

Reply:
I've updated it to use: `kill -SIGHUP $(pidof vault)`

As mentioned in this nicely written support article (https://support.hashicorp.com/hc/en-us/articles/5767318985107-Vault-SIGHUP-Behavior)